### PR TITLE
Refactorings for convert tests

### PR DESCRIPTION
**Pull request by chrzyki**

### CODE REVIEW

File: lingpy/tests/convert/test_graph.py [hunk 1]
```diff
@@ -6,31 +8,32 @@
     nonetworkx = False
 except ImportError:
     nonetworkx = True
+
 try:
-    import igraph as ig
+    import jgraph as ig
```
[the remark this answers is not in the document]
Comment (chrzyki) [reply lingpy/tests/convert/test_graph.py]: My bad, sorry. Was confused by a deprecation warning (that was caused by my local package settings).

[the remark this answers is not in the document]
Comment (LinguList) [reply lingpy/tests/convert/test_graph.py]: no deal, as you see in my summary: we're not really clear on that in the docu so far anyway...

[the remark this answers is not in the document]
Comment (LinguList) [reply lingpy/tests/convert/test_graph.py]: feel free to merge afterwards, and thanks!

[the remark this answers is not in the document]
Comment (chrzyki) [reply lingpy/tests/convert/test_graph.py]: Thanks, too. :) Will make a note about the package installation.

**Pull request merged**